### PR TITLE
Remove deprecated packages and polygon

### PR DESCRIPTION
**Pull request by noramehesz**
ReadME changes are missing, will add this soon.

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -1,67 +1,64 @@
...
-    qs: queryParams || {},
-    headers: {
-      'content-type': 'application/json',
-    },
-    body: body || undefined,
-    json: true,
-  })
+async function makeRequest(f, endpoint, queryParams, body, method) {
+  const resp = await f(endpoint, queryParams, body, method);
+  return resp.data;
```
Comment (Mark-Pintye) [inline lib/api.js:57]: I think sendRequest is a more reasonable name for this function

File: lib/api.js [hunk 2]
```diff
@@ -1,67 +1,64 @@
...
+          }
+        : {
+            "content-type": method !== "DELETE" ? "application/json" : "",
+            "APCA-API-KEY-ID": keyId,
+            "APCA-API-SECRET-KEY": secretKey,
+          },
     body: body || undefined,
-    json: true,
-  })
+  };
```
Comment (Mark-Pintye) [inline lib/api.js:48]: Please use data there aswell like in the function above

**Review by Mark-Pintye — DISMISSED** [review]
Looks good to me

File: README.md [hunk 3]
```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
...
-    Channels for the Polygon service: `['T.*', 'Q.*', 'A.*', 'AM.*']`.<br>
-    Channels for the Alpaca data service: `['alpacadatav1/T
-    .*', 'alpacadatav1/Q.*', 'alpacadatav1'/AM.*]`
+* `client.onConnect(function() {})`: all the following code will be executed after 
+  the user authentication. You can also subscribe and unsubscribe for data 
+  outside this function.
+* `websocket.subscribeForTrades(["symbol"])`: Subscribe for trades data for the
+  given symbols(s). You can do the same with quotes, bars, dailyBars, statuses
+  and lulds.
+* `websocket.onStockTrade(function(trade) {})`: Get the data and do wathever you
```
Comment (Mark-Pintye) [inline README.md:417]: wathever -> whatever. This sentence sounds a bit weird though

File: README.md [hunk 4]
```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
...
-    You need to specify the channel you want to
-   subscribe to as specified here:<br>
-    Channels for the Polygon service: `['T.*', 'Q.*', 'A.*', 'AM.*']`.<br>
-    Channels for the Alpaca data service: `['alpacadatav1/T
-    .*', 'alpacadatav1/Q.*', 'alpacadatav1'/AM.*]`
+* `client.onConnect(function() {})`: all the following code will be executed after 
+  the user authentication. You can also subscribe and unsubscribe for data 
+  outside this function.
+* `websocket.subscribeForTrades(["symbol"])`: Subscribe for trades data for the
+  given symbols(s). You can do the same with quotes, bars, dailyBars, statuses
```
Comment (Mark-Pintye) [inline README.md:415]: symbols(s) -> symbol(s)

File: README.md [hunk 5]
```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
...
-  the Polygon server.<br>
-  Please note that Polygon and Alpaca servers use different channels. <br>
-    You need to specify the channel you want to
-   subscribe to as specified here:<br>
-    Channels for the Polygon service: `['T.*', 'Q.*', 'A.*', 'AM.*']`.<br>
-    Channels for the Alpaca data service: `['alpacadatav1/T
-    .*', 'alpacadatav1/Q.*', 'alpacadatav1'/AM.*]`
+* `client.onConnect(function() {})`: all the following code will be executed after 
+  the user authentication. You can also subscribe and unsubscribe for data 
+  outside this function.
```
Comment (Mark-Pintye) [inline README.md:413]: outside this function -> outside of this function

File: README.md [hunk 6]
```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
...
+* `websocket.subscribeForTrades(["symbol"])`: Subscribe for trades data for the
+  given symbols(s). You can do the same with quotes, bars, dailyBars, statuses
+  and lulds.
+* `websocket.onStockTrade(function(trade) {})`: Get the data and do wathever you 
+  want with it inside this function.
+* `websocket.unsubscribeFromTrades(["symbol"])`: Unsunscribe from symbol(s).
+* `websocket.onDisconnect(function() {})` and `websocket.disconnect()`: the function
+  inside the onDisconnect will run when you cann disconnect, then closes the connection
```
Comment (Mark-Pintye) [inline README.md:421]: I think `cann` should be omited here

File: README.md [hunk 7]
```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
...
+  between the client and server.
+<br><br>
```
Comment (Mark-Pintye) [inline README.md:423]: I think these wont create breaklines, maybe this: `<br/><br/>`

Comment (noramehesz) [reply README.md:423]: It works

**Review by Mark-Pintye — APPROVED** [review]

**Pull request merged**